core/trino-parser/src/main/java/io/trino/sql/tree/DropRole.java

```diff
@@ -55,6 +57,11 @@ public Optional<Identifier> getCatalog()
         return catalog;
     }
 
+    public boolean isIfExists()
```

Comment:
`isExists`, for consistency with all the other classes (DropTable, DropView, DropSchema, etc)

Reply:
Done.